.expeditor/create-manifest.rb

```diff
@@ -300,7 +300,7 @@ def parse_products_meta_file(path)
   # while still allowing the clients of these databases to upgrade their client
   # libraries if any fixes are shipped there.
   "automate-postgresql"    => {"origin" => "chef", "name" => "automate-postgresql",    "version" => "9.6.11", "release" => "20200723182209"},
```

Comment:
Do we neeed to do postgresql as well?

Reply:
let me double check

Reply:
the package was modified, so i brought it in